source/index.html.md

```diff
@@ -373,10 +376,341 @@ The details endpoint provides a report of the campaigns for the agent in the tok
 }
 ```
 
-## Update Listing Endpoint
+### Update Listing Endpoint
 
 This will update the given summary based on the provided mls_id and zip with the other provided params. Right now any agent id passed into the token will work to update any listing, it doesn't have to be the agent that created the listing. The customer id has to be the same as the one that created the listing though.
 
 All of the values that are used to create the summary can be updated, only exceptions are mls_id and zip which are used to determine which listing you want to change.
 
 > http://boost.homespotter.com/dashboard/integration/update/$partner_short_code?token=$token&mls_id=785773599&zip=55401&price=99999
+
+
+# Boost Integration V2
```

Comment:
Can we shift this above V1, so that it stands out as the recommended way to do this integration?

Reply:
@levmar Change is made.